### DISCUSSION TITLE
Trouble building DLLs for VS2019

### DISCUSSION THREAD

**Discussion by mfellows** [coin-or/coinbrew#67] [Q&A] [answered]
I have a project that depends on a few libraries that are mainly built using vcpkg (i.e. vcpkg install boost:x64-windows), which by default work with Release mode and the -MD flag, but I've been struggling to get coinbrew to give me a compatible build - by default it seems to build static Debug (-MTd) libs which eventually throw a linker error when I try to build the project that depends on everything.

I've tried adding the --enable-shared flag to the coinbrew command line by itself, and also with ADD_CXXFLAGS="-MD -LD", but it seems to only change the file size of the resulting libs without building a corresponding set of dlls.

It must be user error, but I'm out of things to try - any help would be much appreciated! basically just trying to get "plain old" Release-mode dlls that play nice with other libraries built with vcpkg.

Using Windows 10, Msys2, and VS2019, following these steps:

- install msys2
- install the Intel oneAPI base toolkit + HPC toolkit (Fortran compiler and MKL)
- at the command prompt:
        set PATH=C:\development\msys64\usr\bin;%PATH%
        "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"
        c:\development\msys64\usr\bin\bash.exe
        pacman -S make wget tar patch pkg-config diffutils dos2unix git svn
        mkdir /coin-or
        cd /coin-or
        wget https://raw.githubusercontent.com/coin-or/coinbrew/master/coinbrew
        chmod a+x ./coinbrew
        ./coinbrew fetch Cbc --latest-release --no-prompt
        ./coinbrew build Cbc ADD_CXXFLAGS="-MD -LD" --enable-shared --parallel-jobs=4 --enable-msvc --no-prompt --build=x86_64-w64-mingw32 --tests=none --prefix=/c/Development/lib/Cbc

**Comment by tkralphs** [comment, marked as answer]
`--enable-msvc=MD` should be what you need. My apologies if that's not mentioned in the documentation anywhere!
